src/content/developers/docs/ethereum-stack/index.md

```diff
@@ -33,7 +33,7 @@ As a dapp developer, you'll need to write smart contracts only if you want to ad
 
 In order for an application to interact with the Ethereum blockchain, it must connect to an [Ethereum node](/developers/docs/nodes-and-clients/). Connecting to a node allows you to read blockchain data and/or send transactions to the network.
 
-Ethereum nodes are computers running software - an Ethereum client. A client is an implementation of Ethereum that verifies all transactions in each block, keeping the network secure and the data accurate. Ethereum nodes ARE the Ethereum blockchain. They collectively store the state of the Ethereum blockchain and reach consensus on transactions to mutate the blockchain state.
+Ethereum nodes are computers running software - an Ethereum client. A client is an implementation of Ethereum that verifies all transactions in each block, keeping the network secure and the data accurate. Ethereum nodes are the Ethereum blockchain. They collectively store the state of the Ethereum blockchain and reach consensus on transactions to mutate the blockchain state.
```

Comment:
I think this was deliberate to add emphasis here. However, thinking about translations (cc: @lukassim) I don't think it makes a ton of sense to keep it this way. But I also don't think removing the emphasis altogether is correct. What about if we wrap this sentence with bold tags?

```suggestion
Ethereum nodes are computers running software - an Ethereum client. A client is an implementation of Ethereum that verifies all transactions in each block, keeping the network secure and the data accurate. **Ethereum nodes are the Ethereum blockchain**. They collectively store the state of the Ethereum blockchain and reach consensus on transactions to mutate the blockchain state.
```

Reply:
Looking at the TM, most translators have kept this in uppercase, only a handful of translators have changed it to lowercase.
Bold tags should work fine in Crowdin, so I guess either option would be okay for the translations.